events/messageCreate.js

```diff
@@ -1,13 +1,82 @@
 const { Events } = require('discord.js');
 
+const fs = require('fs');
+
+let response_common = [];
+let response_rare = [];
+let response_epic = [];
+let response_legendary = [];
+
+function fetchMessages() {
+  fs.readFile('response_common.txt', 'utf8', (err, data) => {
+    if (err) {
+      console.error('There was a problem reading the file:', err);
+      return;
+    }
+	response_common = data.split('\n');
+    // You can perform further operations with the 'messages' array here
+  });
+
+  fs.readFile('response_rare.txt', 'utf8', (err, data) => {
+    if (err) {
+      console.error('There was a problem reading the file:', err);
+      return;
+    }
+	response_rare = data.split('\n');
+    // You can perform further operations with the 'messages' array here
+  });
+
+  fs.readFile('response_epic.txt', 'utf8', (err, data) => {
+    if (err) {
+      console.error('There was a problem reading the file:', err);
+      return;
+    }
+	response_epic = data.split('\n');
+    // You can perform further operations with the 'messages' array here
+  });
+
+  fs.readFile('response_legendary.txt', 'utf8', (err, data) => {
+    if (err) {
+      console.error('There was a problem reading the file:', err);
+      return;
+    }
+	response_legendary = data.split('\n');
+    // You can perform further operations with the 'messages' array here
+  });
+}
+
+// Call the function to fetch messages
+fetchMessages();
+  
+// Function to get a random message
+function getRandomMessage() {
+	let randomNumber = Math.floor(Math.random() * 100);
+	console.log("CONSOLE: number is" + randomNumber)
+	if (randomNumber <= 80) {
```

Comment:
We should not introduce magic number here See: https://en.wikipedia.org/wiki/Magic_number_(programming)
Using a constant on top of the file could be better. Or even better using a config file to store the probability, the filenames, etc; ideally only loaded once. We split it to another ticket. Let me know what you think!